frog/imports/client/SingleActivity/index.js

```diff
@@ -263,22 +44,37 @@ class SingleActivity extends React.Component<PropsT, StateT> {
           <ConfigPanel
             activityType={activityType}
             classes={classes}
-            onSubmit={conf =>
-              this.setState({ stage: this.state.stage + 1, activity: conf })
-            }
+            onSubmit={conf => {
+              this.setState({ stage: this.state.stage + 1, activity: conf });
+              Meteor.call(
+                'create.graph.from.activity',
+                conf.activityType,
+                conf.config,
+                3,
+                (err, res) => {
+                  if (err) {
+                    // eslint-disable-next-line no-console
+                    console.log(err);
```

Comment:
We should throw instead of logging as this puts the application in a non-deterministic state.

Reply:
Better to show error message and let user try again? Currently it gets stuck on the loading screen.

Reply:
Should I display a Modal in this case?